File: Linear/Static/Array/02_Inserting/End/Built-In/Inserting.py
arr = [10, 20, 30, 40]
x = 50
print("Array before insertion")
for i in range(len(arr)):
    print(arr[i], end=" ")

# Insert x at the end.
arr.append(x)

print("\nArray after insertion")
for i in range(len(arr)):
    print(arr[i], end=" ")

File: Linear/Static/Array/02_Inserting/End/Built-In/Inserting.java
import java.util.ArrayList;
import java.util.Arrays;

class InsertingEndBuiltIn {
    public static void main(String[] args) {
        ArrayList<Integer> arr = new ArrayList<>(Arrays.asList(10, 20, 30, 40));
        int x = 50;
        System.out.println("Array before insertion");
        for (int i = 0; i < arr.size(); i++) {
            System.out.println(arr.get(i) + " ");
        }

        // Insert x at the end.
        arr.add(x);

        System.out.println("\nArray after insertion");
        for (int i = 0; i < arr.size(); i++) {
            System.out.println(arr.get(i) + " ");
        }
    }
}
